app/services/item_service.rb

```diff
@@ -0,0 +1,11 @@
+class ItemService
+  def self.conn
+    conn = Faraday.new(url: "http://localhost:3000")
+  end
```

Comment:
`self.conn` is exactly the same in `MerchantService`. This could be DRY'd up by combining `ItemService` and `MerchantService` into one class, which I think you mentioned in your submission. 
Alternatively, maybe they could both inherit this behavior.